### ISSUE TITLE
您好，我想问这种方法是通过训练代码对wsj02_mix里所有混合语音进行训练，再在测试代码里输入wsj02_mix里的一个混合语音得到该混合语音的分离结果吗？

### ISSUE THREAD

**Issue by shinyruoupup** [JusperLee/Dual-Path-RNN-Pytorch#12]

**Comment by JusperLee** [comment]
Dual-Path-RNN belongs to the open-setting method, so the training set and test set of this method do not overlap (the speakers are not repeated). The separation process is to input the audio of the mix (mix voice belongs to WSJ0-2mix) to separate the audio of the two speakers.

**Comment by shinyruoupup** [comment]
还是有点没明白训练，测试，分离三者的关系，测试和分离是否是同一个步骤？
训练代码train_rnn.py输入的训练集语音是WSJ0-2mix里单个人说话的语音？分离语音的过程就是在测试代码dualrnn_test.py里吗？
分离输入的混合语音是WSJ0-2mix自带的，还是需要我们自己人工合成？

**Comment by JusperLee** [comment]
1. Testing and training are separate voices.
2. The training input is mixed speech (in general, we are talking about the mixed speech of two speakers).
3. For the pre-trained model, we can use dualrnn_test.py for speech separation.
4. The input mixed speech is synthesized manually. WSJ0-2mix is ​​manually mixed through the WSJ0 data set.

**Comment by shinyruoupup** [comment]
首先WSJ0人工合成为混合的WSJ0-2mix，然后WSJ0-2mix 的一部分用于训练，一部分用于分离（两者不重叠）是这个意思吗？

**Comment by JusperLee** [comment]
You can understand that.

**Comment by JusperLee** [comment]
This problem occurs in deep learning, which may be related to data distribution.

**Comment by JusperLee** [comment]
The timeout is automatically closed.
